openmc/material.py

```diff
@@ -407,6 +407,24 @@ def remove_nuclide(self, nuclide):
             if nuclide == nuc.name:
                 self.nuclides.remove(nuc)
 
+    def remove_element(self, element):
+        """Remove an element from the material
+
+        Parameters
+        ----------
+        element : str
+            Element to remove
+
+        """
+        cv.check_type('element', element, str)
+
+        # If the Material contains the element, delete it
+        for nuc in reversed(self.nuclides):
+            element_name = re.split(r'(\d+)', nuc.name)[0]
+            print(element_name, element)
```

Comment:
Capture group in the regex is not needed (and probably adds overhead) -- better to get rid of it
```suggestion
            element_name = re.split(r'\d+', nuc.name)[0]
```